Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Person
+    {
+        private readonly string _name;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_name' or refactor the code to use its value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)